src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+form {
+  margin-top: 20px;
+}
+
+.input1 {
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 25px 26px;
+  font-family: inherit;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+}
+
+.input2 {
```

Comment:
```suggestion
.input--small {
```

Reply:
looks like `input1` and `input2` have repeated logic; so leave class `input1` as `input`, that will have all styles for big input, and change `input2`  to `input--small`, in `input--small` leave all styles that are different for `input` (checklist p2)